### DISCUSSION TITLE
Edit Species Form Not Showing in Modal

### DISCUSSION THREAD

**Discussion by thedigitalmenagerie** [nss-evening-cohort-14/evening-client-side#198] [Help Me!] [answered]
Honey-Rae Swan

Have you searched the other help tickets? 
Yes. 

Describe your issue
My editSpeciesForm() will not show up in the form modal. This is working for add and the title of the modal form is changing but I am unsure of how these errors are related or why these are being viewed as null. 

List any error messages that appear
editSpecies.js:4 Uncaught (in promise) TypeError: Cannot read property 'name' of null
    at editSpeciesForm (editSpecies.js:4)
    at eval (domEvents.js:44)

What have you googled?
https://salesforce.stackexchange.com/questions/261605/how-to-refresh-contents-of-modal-on-button-click-vf-page
(I'm not really sure what else to google)

What have you tried? Tell us everything you have tried.
I tried just calling the editSpeciesForm instead of the one for the specific species card and it also did not show up then. It is working for add. My team and I could not find differences between the two that we could identify as causing the issue. 

Copy/paste link TO THE FILE on your branch or in your repo that you are having issues with NO SCREEN SHOTS 🥰
https://github.com/nss-evening-cohort-14/nautilus-explorer-dashboard-nautilus-explorer-dashboard/tree/hr-updateSpecies

**Comment by drteresavasquez** [comment, marked as answer]
@thedigitalmenagerie it means that you don't have something with the key of 'name'. 

Did you get this? Sorry I did not see the ticket sooner.

**Reply by thedigitalmenagerie** [reply]
I haven't but somehow changed the problem to it still showing addForm in the module since then! hahah

**Reply by thedigitalmenagerie** [reply]
and that's ok!

**Reply by drteresavasquez** [reply]
If you haven't gotten it by class on Monday night, let's jump on and work through the issue in class.

**Reply by thedigitalmenagerie** [reply]
I kept typing firebasKey in the edit modal interpolations. JS didn't know what the heck I was telling it to do.
